### ISSUE TITLE
Wrong alignment of on-focus indicator for filter tabs

### ISSUE THREAD

**Issue by AyanChoudhary** [creativecommons/cccatalog-frontend#1142]
## Description

The on-focus indicator for the close button in filter-tabs is placed incorrectly. **Note:** This should be fixed by implementing the [new `removable tag` component in vocabulary](https://cc-vocabulary.netlify.app/?path=/story/elements-buttons--removable-tag).

## Reproduction

1. Use the keyboard to navigate to the filter tabs.
2. Press tab to focus on the x button
3. See error.

## Screenshots

![CC Search - Brave 05-08-2020 18_50_48](https://user-images.githubusercontent.com/45792768/89419711-5237d380-d74f-11ea-8cc9-7de55fbe2f03.png)

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by neeraj-2** [comment]
Can i work on this issue

**Comment by AyanChoudhary** [comment]
Yes go on ahead @neeraj-2

**Comment by neeraj-2** [comment]
![allignment_after_change](https://user-images.githubusercontent.com/56186425/90307704-771e0a80-def6-11ea-8c85-b71666fa12c3.PNG)
I have attached screenshot ,after changing code
